examples/minimal.cdl

```diff
@@ -0,0 +1,220 @@
+netcdf sp041_20191205T1757 {
```

Comment:
As per OG conventions, I think the filename should be something like sp041_20191205T1757_delayed.nc